datasafe-business-api/src/main/java/de/adorsys/datasafe/business/api/encryption/cmsencryption/CMSEncryptionService.java

```diff
@@ -8,12 +8,15 @@
 
 import de.adorsys.datasafe.business.api.deployment.keystore.types.KeyID;
 import de.adorsys.datasafe.business.api.deployment.keystore.types.KeyStoreAccess;
+import org.bouncycastle.asn1.ASN1ObjectIdentifier;
```

Comment:
I thin we can inject algorithm configuration using dagger - i.e. create some EncryptionConfig interface with default implementation and user will decide what to use

Reply:
Ok, good idea. What do you think, just an algorithm filed is enough for this config, or something other should be also in config?

Reply:
I think we can start with something simple like just algorithm field